client/web/compose/src/views/Public/Pages/Records/View.vue

```diff
@@ -270,6 +271,10 @@ export default {
       const { recordID } = this.record || {}
       return this.getNextAndPrevRecord(recordID)
     },
+
+    viewHasBack () {
+      return this.previousPages.length > 0 || this.modalPreviousPages.length > 0
```

Comment:
If its in module then one is useful if no the other is, this way you can have false positives.